### PR TITLE
Mark `OnceState::poison` as `pub`

### PR DESCRIPTION
**Pull request by caass**
tracking issue: #130327

### CODE REVIEW

File: library/std/src/sync/once.rs [hunk 1]
```diff
@@ -365,9 +381,9 @@ impl OnceState {
     }
 
     /// Poison the associated [`Once`] without explicitly panicking.
-    // NOTE: This is currently only exposed for `OnceLock`.
+    #[unstable(feature = "once_poison_pub", issue = "130327")]
```
Comment (GrigorenkoPV) [inline library/std/src/sync/once.rs:384]: The tracking issue seems to have a different name for the feature

**Comment by Amanieu** [comment]
Please fix the feature name either here or in the tracking issue.

@rustbot author

**Comment by alex-semenyuk** [comment]
@caass 
From wg-triage. Do you have updates on this PR

**Comment by Dylan-DPC** [comment]
Closing this as inactive. Feel free to reöpen this pr or create a new pr if you get the time to work on this. Thanks

**Pull request closed without merging**